index.rst

```diff
@@ -59,8 +64,8 @@ The RMS was calculated with a rolling of 10 (50 rolling is approximately 1 s of
 Results
 ---------
 
-.. list-table:: # Events with RMS greater than required in the first 3 s
-   :widths: 25 25 50
+.. list-table:: Events with RMS greater than required in the first 3 s
```

Comment:
I have a question about the table. Does the "All" row indicate that every position exceeded the rms limit, or does it mean that at least one position went over the limit?